client/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+ import { __ } from '@wordpress/i18n';
+ import { addFilter } from '@wordpress/hooks';
+
+import OnboardingWizard from './additional-methods-setup/onboarding-wizard.js';
+
+addFilter(
+	'woocommerce_admin_pages_list',
+	'woocommerce-gateway-stripe',
+	( pages ) => {
+		pages.push( {
+			container: OnboardingWizard,
+			path: '/onboarding',
```

Comment:
I just realized - maybe just `/onboarding` could create some issues with scoping - what do you think about `/stripe/onboarding`?

Reply:
No longer using the React-powered page setup.

Reply:
FWIW, I noticed that the layout is slightly different - we'll need to fix that as well. Should we do it on a separate PR?

Reply:
Yes! +1 on another PR.